Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org' do
+  gem 'sinatra'
+  gem 'rspec'
+end
```

Comment:
Just realised you've provided a Gemfile, but I think you don't need it. Your install instructions don't reference bundler and there are no specs. It's an unusual case in that installing a single gem does everything you need, so you could either go for dropping the Gemfile, or else use it, and make your install instructions refer to it. 

All of these comments might be regarded as OTT in the context of a tiny example app :)